valgrind:Add "3.0.*" statement to the "case" statement for supporting the kernel 3.0.* .

Signed-off-by: Tong Lin <tong.lin@intel.com>

Upstream-Status: Pending

diff --git a/configure.in b/configure.in
index 3878619..f09720e 100644
--- a/configure.in
+++ b/configure.in
@@ -229,10 +229,13 @@ case "${host_os}" in
         	    AC_MSG_RESULT([2.4 family (${kernel})])
         	    AC_DEFINE([KERNEL_2_4], 1, [Define to 1 if you're using Linux 2.4.x])
         	    ;;
-
+             3.0.*)
+                    AC_MSG_RESULT([3.0 family (${kernel})])
+                    AC_DEFINE([KERNEL_3_0], 1, [Define to 1 if you're using Linux 3.0.x])
+                    ;;
              *) 
         	    AC_MSG_RESULT([unsupported (${kernel})])
-        	    AC_MSG_ERROR([Valgrind works on kernels 2.4, 2.6])
+        	    AC_MSG_ERROR([Valgrind works on kernels 2.4, 2.6, 3.0])
         	    ;;
         esac
 
